### PR TITLE
replace quorum checking mechanism with hash comparison.

### PR DESCRIPTION
**Pull request by PawelPawelec-RDX**

### CODE REVIEW

File: src/RadixDlt.NetworkGateway.DataAggregator/ServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -142,6 +143,9 @@ private static void AddTransientApiReaders(IServiceCollection services, out IHtt
         services.TryAddTransient<ITransactionStreamReader, TransactionStreamReader>();
         services.TryAddTransient<INetworkConfigurationReader, NetworkConfigurationReader>();
         services.TryAddTransient<INetworkStatusReader, NetworkStatusReader>();
+
+        services.TryAddTransient<Func<ITransactionStreamReader>>(provider => provider.GetRequiredService<ITransactionStreamReader>);
+        services.TryAddTransient<Func<INetworkStatusReader>>(provider => provider.GetRequiredService<INetworkStatusReader>);
```
Comment (krzlabrdx) [inline src/RadixDlt.NetworkGateway.DataAggregator/ServiceCollectionExtensions.cs:148]: Wait, what? Why do we register factory funcs(?) in DI container?

File: src/RadixDlt.NetworkGateway.DataAggregator/ServiceCollectionExtensions.cs [hunk 2]
```diff
@@ -153,7 +157,7 @@ private static void AddNodeInitializers(IServiceCollection services)
     private static void AddNodeWorkers(IServiceCollection services)
     {
         // Add node workers - these will be instantiated by the NodeWorkersRunner.cs.
-        services.TryAddScoped<INodeWorker, NodeTransactionLogWorker>();
+        services.TryAddScoped<INodeWorker, NodeTransactionFetcherWorker>();
```
Comment (krzlabrdx) [inline src/RadixDlt.NetworkGateway.DataAggregator/ServiceCollectionExtensions.cs:160]: `NodeTransactionFetchWroker`?

File: src/RadixDlt.NetworkGateway.DataAggregator/Services/LedgerTransactionsProcessor.cs [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+        _ledgerConfirmationOptionsMonitor = ledgerConfirmationOptionsMonitor;
+        _networkOptionsMonitor = networkOptionsMonitor;
+        _systemStatusService = systemStatusService;
+        _ledgerExtenderService = ledgerExtenderService;
+        _observers = observers;
+        _clock = clock;
+        _fetchedTransactionStore = fetchedTransactionStore;
+        _topOfLedgerCache = topOfLedgerCache;
+
+        _observers.ForEach(x => x.ResetQuorum());
```
Comment (krzlabrdx) [inline src/RadixDlt.NetworkGateway.DataAggregator/Services/LedgerTransactionsProcessor.cs:126]: Shouldn't we get rid of that?

File: src/RadixDlt.NetworkGateway.PostgresIntegration/CommonDbContext.cs [hunk 4]
```diff
@@ -190,7 +190,7 @@ protected override void ConfigureConventions(ModelConfigurationBuilder configura
             .Properties<TokenAmount>()
             .HaveConversion<TokenAmountToBigIntegerConverter>()
             .HaveColumnType("numeric")
-            .HavePrecision(1000);
+            .HavePrecision(1000, 0);
```
[the remark this answers is not in the document]
Comment (PawelPawelec-RDX) [reply src/RadixDlt.NetworkGateway.PostgresIntegration/CommonDbContext.cs:193]: mistake, reverted!

**Pull request merged**